### ISSUE TITLE
Keeping Current values

### ISSUE THREAD

**Issue by pargit** [TomerFi/home-assistant-custom-components#30]
hi,
how can i keep track of the electric current sensor?
i want to generate a csv/graph of electric usage from the switcher.
thanks
mor

**Comment by TomerFi** [comment]
Yes, You can add the component to the [Recorder](https://www.home-assistant.io/components/recorder/) in order to save its history.
You can also use the [History Graph](https://www.home-assistant.io/components/history_graph/) to monitor the usage and base automatons on.

As far as the csv file, I haven't tried it, but I'm sure this can be achieved manually from the database.
